src/encord_active/lib/model_predictions/writer.py

```diff
@@ -268,48 +344,68 @@ def append_object_label(du_hash: str, frame: int, o: dict, width: int, height: i
 
         for lr in tqdm(self.project.label_rows.values(), desc="Preparing labels", leave=True):
             data_type = lr["data_type"]
+            answers = lr["classification_answers"]
             for du_hash, du in lr["data_units"].items():
                 height = int(du["height"])
                 width = int(du["width"])
 
-                if "im" in data_type:  # img_group or image
+                if data_type in ["image_group", "image"]:
                     frame = int(du["data_sequence"])
-                    for obj in du["labels"]["objects"]:
-                        append_object_label(du_hash, frame, obj, width, height)
+                    for label in du["labels"]["objects"]:
```

Comment:
```suggestion
                    for label in du["labels"].get("objects", []):
```

Reply:
I think the keys are always there, but good point, will change for safety 👍